### PR TITLE
docs(i18n): New Crowdin translations

### PR DESCRIPTION
**Pull request by matchai**
Enjoy!

### CODE REVIEW

File: docs/ja/guide/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+../../README.ja.md
```
Comment (cappyzawa) [inline docs/ja/guide/README.md]: Is this file translated??

Comment (matchai) [reply docs/ja/guide/README.md]: Yes. It looks like it correctly updated the symlink to the new path when I run it locally.

**Comment by matchai** [comment]
I'm currently talking to the Crowdin support team to find out the cause of the formatting issue.

**Comment by matchai** [comment]
Closing this PR in favor of a new one with the `md` parsing fixes done by Crowdin.

**Pull request closed without merging**